sys/net/gnrc/sock/gnrc_sock.c

```diff
@@ -86,6 +86,7 @@ ssize_t gnrc_sock_recv(gnrc_sock_reg_t *reg, gnrc_pktsnip_t **pkt_out,
             if (msg.content.value == _TIMEOUT_MAGIC) {
                 return -ETIMEDOUT;
             }
+            __attribute__((fallthrough));
```

Comment:
I've tried several forms of the fallthrough comment, including ` /* Falls Through. */`, `/* falls through */` etc, and they all (including `/* FALLTHROUGH */`) work only after `#endif`.